### ISSUE TITLE
Inspector doesn't call correctly "Error: Unexpected token < in JSON at position 0"

### ISSUE THREAD

**Issue by NumericPrime** [dwavesystems/dwave-inspector#128]
When I tried the open_problem method wich the block values True and False the first one worden once but the I noticed  that no matter in wich script I called the Show or open_problem funktioniert it always loaded the inspector but only showing the message:

Error: Unexpected token < in JSON position 0

The same problem was discussed a year ago #46 and is discussed in #106. But none of the sollutions worked.

I tried:
reinstall the library
the sollution discussed in #46 

Since it happens everytime when I start the inspector I can't send and specific code to reproduce the error.

**Comment by NumericPrime** [comment]
Update: Turns out restarting the computer did help.
Also open_problem in general seems to trigger the problem. So how can I use dwave.inspector.open_problem(SAPI)
without triggern the inspector to faul until restarting the computer.

**Comment by randomir** [comment]
Hi @NumericPrime,

Can you reproduce this issue on your system/environment with [one of the examples](https://github.com/dwavesystems/dwave-inspector/tree/master/examples)?

**Comment by NumericPrime** [comment]
Sure
` import dwave.inspector
dwave.inspector.open_problem('b6b7e962-ea76-442d-b6fc-ebc35177f9b4')  `

**Comment by NumericPrime** [comment]
The problem occurs at all three examples the problem ist triggered in the inspecting step. The browser-tab got opened only showimg the error message.
The script is never finished though.

**Comment by NumericPrime** [comment]
Is the open_problem method even supposed to be called directly.

Maybe the error is caused by me not providing suitable data causing this error and maybe messing up my Instanzen of the app_server. Since the script never finishes maybe this carries over to every other instance?

**Comment by randomir** [comment]
So, just to clarify -- you run one of the examples _verbatim_, without any code modifications, and you see this error? I'm asking because you mentioned calling `open_problem`. For general use, only `dwave.inspector.show()` should be used.

**Comment by NumericPrime** [comment]
Yes I ran the code without modifications exept letting it print a number at the end to Check that the problem really never finishes. And it gives this error, if the bug has been triggered. 
The bug gets triggered, when I use the open_problem function, with block set to true and no timeout specified, showing the error message.
The problem is once the bug has been triggered it shows this message every instance of me opening show.
If I did specify a timeout, it will still show the error but it doesn't affect all other instences of show.
